spec/support/participants/practice_helper.rb

```diff
@@ -221,3 +223,21 @@ def reappraisal_2
     created_at: Time.now - (2 * 60 * 60)
   )
 end
+
+def profile
+  @profile ||= Participants::SocialNetworkingModules::Profile.new(
```

Comment:
since these are named the same as the other new ones and we're using a conditional equals, the first one loaded is the object that is being used but we don't want that as we are passing in different arguments. Just remove `@profile ||= ` from each of the new ones. That should take care of it.